src/ontology/mondo-ingest-odk.yaml

```diff
@@ -27,6 +27,8 @@ components:
   products:
     - filename: doid.owl
       source: http://purl.obolibrary.org/obo/doid.owl
+    - filename: gard.owl
```

Comment:
@matentzn I just ran `sh run.sh make update_repo`, but there were more things in the diff than I expected, so I chose not to commit them now and ask first what you'd like me to do.

a. Since some of these changes are not GARD related, I can run this first on `main`, commit there, rebase the `gard` branch w/ those changes, and then run `update_repo` again on that branch and commit those changes.
b. Just run it once on the `gard` branch and commit there
c. Do nothing for now

4 files were updated, and here's the diff: [update_repo diff.txt](https://github.com/monarch-initiative/mondo-ingest/files/11482663/update_repo.diff.txt)

Reply:
Went with 'a'!